playbook-website/lib/pb_doc_helper.rb

```diff
@@ -27,33 +27,58 @@ def nav_hash_array(link)
 
     # Deal with lists of kits, used in Playbook doc and Externally
     # rubocop:disable Style/StringConcatenation
-    def pb_kits(type: "rails", limit_examples: false, dark_mode: false, method: get_kits)
+    def pb_kits(type: "rails", limit_examples: false, dark_mode: false)
       display_kits = []
-      kits = method
+      kits = get_kits(type)
       kits.each do |kit|
-        if kit.is_a?(Hash)
-          nav_hash_array(kit).each do |sub_kit|
-            display_kits << render_pb_doc_kit(sub_kit, type, limit_examples, false, dark_mode)
-          end
-        else
-          display_kits << render_pb_doc_kit(kit, type, limit_examples, false, dark_mode)
+        nav_array = nav_hash_array(kit)
+        next unless nav_array.is_a?(Array)
+
+        nav_array.each do |sub_kit|
+          display_kits << render_pb_doc_kit(sub_kit, type, limit_examples, false, dark_mode)
         end
       end
       raw("<div class='pb--docItem'>" + display_kits.join("</div><div class='pb--docItem'>") + "</div>")
     end
     # rubocop:enable Style/StringConcatenation
 
-    # rubocop:disable Naming/AccessorMethodName
-    def get_kits
+    def get_kits(type = "rails")
       menu = YAML.load_file(Playbook::Engine.root.join("dist/menu.yml"))
-      menu["kits"]
+      menu["kits"][type]
     end
 
-    def get_kits_pb_website
-      menu = YAML.load_file(Rails.root.join("config/menu.yml"))
-      menu["kits"]
+    def aggregate_kits
+      menu = YAML.load_file(Playbook::Engine.root.join("dist/menu.yml"))
```

Comment:
@jasperfurniss  Any idea why we are not using this? Might be the wrong path but this is called once and available everywhere. For some reason we stopped referencing this. I'll bet this would save some server processing if we wen't back to it.

https://github.com/powerhome/playbook/blob/master/playbook-website/config/initializers/global_variables.rb#L3-L4

Reply:
@nickamantia  Yeah let's go ahead and reference those cached versions that @jasoncypret pointed out from the global_variables.rb. Let me know if you want syntax help change these lines.